qoqo_qiskit/docs/conf.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+"""Configuration of sphinx documentation module"""
+try:
+    with open("../qoqo_qiskit/__version__.py") as f:
+        lines = f.readlines()
+    version = lines[-1].strip().split("'")[1].strip()
+except Exception:
+    version = "0.0.0"
+versions = version.split(".")
+main_version = "{}.{}".format(versions[0], versions[1])
```

Comment:
We should now extract the version number from the pyproject.toml file. There are python toml packages that can easily parse the toml file to a dictionary